ui/app/AppLayouts/Chat/popups/community/AccessExistingCommunityPopup.qml

```diff
@@ -72,35 +66,19 @@ StatusModal {
 
     rightButtons: [
         StatusQControls.StatusButton {
+            id: importButton
+            enabled: false
             //% "Import"
             text: qsTrId("import")
             onClicked: {
-                if (!validate()) {
-                    return;
-                }
-
-                communityKey = keyInput.text.trim();
+                let communityKey = keyInput.text.trim();
                 if (!communityKey.startsWith("0x")) {
                     communityKey = "0x" + communityKey;
                 }
 
-
-                root.error = root.store.chatsModelInst.communities.importCommunity(communityKey, Utils.uuid())
-                if (!!root.error) {
-                    creatingError.text = error;
-                    return creatingError.open();
-                }
-
+                root.store.communitiesModuleInst.importCommunity(communityKey);
```

Comment:
```suggestion
                root.store.importCommunity(communityKey);
```

Reply:
@osmaczko yes, I missed that, if we want to stick to some qml rules we introduced, we have call appropriate function from the store which will do what it needs with the real "module" behind.